subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -950,7 +1044,10 @@ static void isr_rx(void *param)
 
 		start_us = hcto;
 		hcto = radio_tmr_start_us(0U, start_us);
-		LL_ASSERT(hcto == (start_us + 1U));
+		/* FIXME: Assertion check disabled until investigation as to
+		 *        why there is high ISR latency causing assertion here.
+		 */
+		/* LL_ASSERT(hcto == (start_us + 1U)); */
```

Comment:
Is this intended to be merged?

Reply:
At random occasions there is spuriously high radio ISR latency causing the subevent scheduling to be delayed caught in this assertion.
Yes, will get back adding the assertion back in a separate PR when analyzing the assert failure.